app/Store.php

```diff
@@ -8,7 +8,9 @@ class Store extends Model
 {
     function reviews()
     {
-        return $this->hasMany('App\Review');
+        return $this->hasMany('App\Review')
+        ->where('establish_at','<',now())
+        ->get();
```

Comment:
publishですね！w
そしてget() もいらないので消しておきましょう！

```suggestion
        ->where('published_at','<',now());
```

Reply:
なんでそう書いてしまったんでしょう、、、
修正しました！